CovidGuardianClient/lib/PathHelpers.js

```diff
@@ -0,0 +1,17 @@
+
+// return true if a = b ± padding
+export function fuzzyEqual(a, b, padding) {
+  return (a <= b && b - Math.abs(padding) <= a) || (a > b && b + Math.abs(padding) >= a);
```

Comment:
return Math.abs(a-b) <= Math.abs(padding) to simplify a bit?

Reply:
fixed